File: src/AoC16/s16e01-typescript/part2.ts
#!/usr/bin/env ts-node
/** Advent of Code - Part 2 */

import * as fs from "fs";
import { solvePart2 } from "./common";

// Read from stdin
const STDIN_FD = 0;
const data = fs.readFileSync(STDIN_FD, "utf-8").trim();
const result = solvePart2(data);
console.log(result);

File: templates/python/template_python/part2.py
"""Part 2 solution runner."""

import sys

from template_python import solve_part2


def main():
    """Read input from stdin and print part 2 result."""
    input_text = sys.stdin.read()
    result = solve_part2(input_text)
    print(result)


if __name__ == "__main__":
    main()
